cmd/zpool/zpool_main.c

```diff
@@ -4174,12 +4174,31 @@ print_zpool_script_list(void)
 		printf("\n");
 		closedir(dir);
 	} else {
-		fprintf(stderr, gettext("Can't open %s scripts dir\n"),
-		    ZPOOL_SCRIPTS_DIR);
+		fprintf(stderr, gettext("Can't open %s scripts dir\n"), path);
 	}
 }
 
 /*
+ * Print out help text for all zpool status/iostat -c scripts.
+ */
+static void
+print_zpool_script_list(void)
+{
+	const char *hdir;
+	char *userdir;
+
+	hdir = getenv("HOME");
+	if (hdir != NULL) {
+		if (asprintf(&userdir, "%s/.zfs/zpool", hdir) != -1) {
```

Comment:
it sounds unwise to me to use ~/.zfs, especially in this position, when it could conflict with zfs' one to present snapshots and when most of the guides suggest to create a separate dataset for each home directory